### PR TITLE
[CentralDashboard] Docs added for central-dashboard

### LINKED ISSUE #1107: 1.0 Docs for centraldashboard
We currently don't have application docs for the central dashboard.
https://www.kubeflow.org/docs/components/

We should have docs for each application that goes 1.0. So we will need some documentation for centraldashboard. This can probably be pretty simple since the central dashboard is pretty self explanatory.

We do have this documentation about accessing UIs
https://www.kubeflow.org/docs/other-guides/accessing-uis/

But its kind of buried.

We might want to move or include some of that content in the central dashboard docs.

### PR DESCRIPTION
**Pull request by avdaredevil**
## About
Kubeflow Central-Dashboard docs added:
- General about for Central Dashboard
- Registration flow docs
- GIF visual for registration flow

### Meta
/area centraldashboard
/area docs
/area front-end
/priority p1
/assign @avdaredevil
/cc @kwasi @sarahmaddox @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1538)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
@avdaredevil When this PR is ready for review, please unassign yourself and assign me.

**Comment by sarahmaddox** [comment]
/hold

Applying a hold until this PR is ready for review.

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1538--competent-brattain-de2d6d.netlify.com/docs/central-dash/

**Comment by jlewi** [comment]
If you look at the preview and look at the left hand side navigation
https://deploy-preview-1538--competent-brattain-de2d6d.netlify.com/docs/central-dash/

i.e. see

Documentation 
   About
   Central Dashboard
   ...
Components of Kubeflow
   ... 
   Central Dashboard
   ...

So the central dashboard docs are appearing in two places. They should probably just be under components of Kubeflow.

@sarahmaddox is that one of the issues you were referring to ?

**Comment by sarahmaddox** [comment]
Huh, I don't know what happened to my comment on the directory structure. As you say, @avdaredevil , the comment has gone missing. Thanks for your comment too @jlewi - that's not the same issue as the one I was pointing out, but is related. I'll start another review now.

File: content/docs/components/central-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
++++
+title = "Central Dashboard"
+description = "All about central dashboard in the Kubeflow Ecosystem"
```
Comment (sarahmaddox) [inline content/docs/components/central-dashboard.md]: It's not really about the ecosystem, but more about the Kubeflow system itself. It'd also be good to use words different from those in the title, so that people who don't understand the title have a different view into what this page is about. How about this for the description:

```
An overview of the Kubeflow user interface (UI)
```

File: content/docs/components/central-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+weight = 10
++++
+
+Your Kubeflow deployment includes Central Dashboard as a key component to provide quick access to all components deployed in your cluster. Some features include:
+- Quick action, pipelines cards, metrics, and high level kubernetes events for an overview of your jobs and cluster in one view
```
Comment (sarahmaddox) [inline content/docs/components/central-dashboard.md]: This list does not render properly. See the preview: https://deploy-preview-1538--competent-brattain-de2d6d.netlify.com/docs/central-dash/

To fix, add empty lines before and after lists.

File: content/docs/central-dash/_index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
++++
```
Comment (sarahmaddox) [inline content/docs/central-dash/_index.md]: Let's put all the central dashboard docs into one place, in the **components** section of the docs. So:

Move this page from `content/docs/central-dash/_index.md` to `content/docs/components/central-dash/_index.md`.

File: content/docs/components/central-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
++++
+title = "Central Dashboard"
```
Comment (sarahmaddox) [inline content/docs/components/central-dashboard.md]: Also, we have an existing page about the Kubeflow UI here:
https://www.kubeflow.org/docs/other-guides/accessing-uis/

I'd recommend merging the content of that existing page into this page. As you'll need to delete the existing page, you should also add a redirect pointing to the new page.

I'm happy to do the merge, if you'd prefer not to do it. In that case, please raise an issue in `kubeflow/website` and assign it to me.

File: content/docs/central-dash/_index.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+title = "Central Dashboard"
+description = "All about central dashboard in the Kubeflow Ecosystem"
+weight = 35
++++
+
+Your Kubeflow deployment includes Central Dashboard as a key component to provide quick access to all components deployed in your cluster. Some features include:
```
Comment (sarahmaddox) [inline content/docs/central-dash/_index.md]: Remove the content from this page. The docs follow the pattern of having an empty `_index.md` file for each section, apart from the front matter at the top of the page, apart from the front matter at the top surrounded by the plus signs. The empty index page causes the doc site to generate a list of pages in the section. See the pattern here: https://raw.githubusercontent.com/kubeflow/website/master/content/docs/fairing/_index.md
which corresponds to this page:
https://www.kubeflow.org/docs/fairing/

(Also, the content here is a duplicate of your page `central-dashboard.md` in this PR.)

File: content/docs/central-dash/registration-flow.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
++++
```
Comment (sarahmaddox) [inline content/docs/central-dash/registration-flow.md]: Let's put all the central dashboard docs into one place, in the **components** section of the docs. So:

Move this page from `content/docs/central-dash/registration-flow.md` to `content/docs/components/central-dash/registration-flow.md`.

File: content/docs/central-dash/registration-flow.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+title = "Registration Flow"
+description = "The view that allows you to set up your first namespace in the cluster"
+weight = 10
++++
+
+Your Kubeflow deployment comes deployed with Kubeflow Central Dashboard. 
+
+This view will show when the cluster is:
+- When in `multi-user` isolation mode:
```
Comment (sarahmaddox) [inline content/docs/central-dash/registration-flow.md]: This list doesn't render. See preview: https://deploy-preview-1538--competent-brattain-de2d6d.netlify.com/docs/central-dash/registration-flow/

File: content/docs/central-dash/registration-flow.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  - The entire cluster has no profile (workgroup / namespace) role-bindings
+
+This guide shows you what this view looks like and how the flow appears.
+
+## Quick guide
+Summary of steps:
+
+1. Follow the [Kubeflow getting-started guide](/docs/started/getting-started/) 
+  to set up your Kubeflow deployment and open the Kubeflow UI.
+1. If the view shows up, you're going to be walked through setting up your workgroup binding
```
Comment (sarahmaddox) [inline content/docs/central-dash/registration-flow.md]: As a reader, I'm not sure what this sentence means. Please would you explain in more detail?

File: content/docs/central-dash/registration-flow.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+## Visual
+![Kubeflow registration flow](/docs/images/registration-flow.gif)
```
Comment (sarahmaddox) [inline content/docs/central-dash/registration-flow.md]: Thanks for adding a visual of the registration flow! For accessibility reasons, and because many people find watching a Gif unhelpful, please explain the steps in text as well.

File: content/docs/central-dash/_index.md (hunk 3 above)
Comment (avdaredevil) [reply content/docs/central-dash/_index.md]: Done!

File: content/docs/central-dash/registration-flow.md (hunk 6 above)
Comment (avdaredevil) [reply content/docs/central-dash/registration-flow.md]: Done

File: content/docs/components/central-dashboard.md (hunk 4 above)
Comment (avdaredevil) [reply content/docs/components/central-dashboard.md]: Done, issue made @ #1549

File: content/docs/central-dash/registration-flow.md (hunk 8 above)
Comment (avdaredevil) [reply content/docs/central-dash/registration-flow.md]: Added a lot more detail, sorry about that

File: content/docs/central-dash/registration-flow.md (hunk 9 above)
Comment (avdaredevil) [reply content/docs/central-dash/registration-flow.md]: Did

File: content/docs/components/central-dash/_index.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
++++
+title = "Central Dashboard"
+description = "All about central dashboard in the Kubeflow Ecosystem"
```
Comment (sarahmaddox) [inline content/docs/components/central-dash/_index.md]: We should remove the word "Ecosystem" here too, as it's not clear what the word means in this context.

Suggested wording:

```
The central user interface (UI) in Kubeflow
```

File: content/docs/components/central-dash/overview.md [hunk 11]
```diff
@@ -1,10 +1,11 @@
...
-description = "All about central dashboard in the Kubeflow Ecosystem"
+description = "An overview of the Kubeflow user interface (UI)"
 weight = 10
 +++
 
 Your Kubeflow deployment includes Central Dashboard as a key component to provide quick access to all components deployed in your cluster. Some features include:
+
 - Quick action, pipelines cards, metrics, and high level kubernetes events for an overview of your jobs and cluster in one view
 - Housing view for all sub services running inside the cluster: `Pipelines`, `Katib`, `Notebooks`, etc.
 - Helps users ramp up their environment with a [registration flow](/docs/central-dash/registration-flow/) (for multi-user isolation, or the first namespace for a shared deployment)
```
Comment (sarahmaddox) [inline content/docs/components/central-dash/overview.md]: Both the links in the last two points are broken.

File: content/docs/components/central-dash/registration-flow.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Follow the [Kubeflow getting-started guide](/docs/started/getting-started/) 
+  to set up your Kubeflow deployment and open the Kubeflow UI.
+1. If the registration flow view shows up (because the conditions above are satisfied):
+   - Then the view will walk you through creating the first workgroup binding for your user or cluster (depending on `isolation-mode`)
+
+## Visual
+![Kubeflow registration flow](/docs/images/registration-flow.gif)
+
+#### Transcription
```
Comment (sarahmaddox) [inline content/docs/components/central-dash/registration-flow.md]: Rather than a description of what happens in the gif, we should give step-by-step instructions of what the user needs to do. For example, tell them to click **Start Setup**, then tell them to do whatever they need to do next, and so on.

I'd recommend that you add the steps in the numbered list that you've already started above the gif, then have the gif as the last item on the page. (Remove the transcription.)

Reasoning: The gif is hard to follow, as it moves fast and the person viewing it needs to watch it multiple times to understand what'g going on. We should therefore offer it only as an additional helpful tool, rather than the main form of instruction.

File: content/docs/components/central-dash/_index.md (hunk 10 above)
Comment (avdaredevil) [reply content/docs/components/central-dash/_index.md]: Sounds good!

File: content/docs/components/central-dash/overview.md (hunk 11 above)
Comment (avdaredevil) [reply content/docs/components/central-dash/overview.md]: Fixed!

File: content/docs/components/central-dash/registration-flow.md (hunk 12 above)
Comment (avdaredevil) [reply content/docs/components/central-dash/registration-flow.md]: I see, lmk if this update is better

**Comment by sarahmaddox** [comment]
Thanks @avdaredevil!

/hold cancel
/lgtm
/approve
